docs/legacy/data-ingestion.asciidoc

```diff
@@ -1,35 +1,35 @@
 [[tune-data-ingestion]]
-== Tune data ingestion
+=== Tune data ingestion
 
 This section explains how to adapt data ingestion according to your needs.
 
-* <<tune-apm-server>>
-* <<tune-es>>
-
-
+[float]
 [[tune-apm-server]]
 === Tune APM Server
 
-++++
-<titleabbrev>APM Server</titleabbrev>
-++++
-
 * <<add-apm-server-instances>>
 * <<reduce-payload-size>>
 * <<adjust-event-rate>>
 
 [[add-apm-server-instances]]
 [float]
-==== Add APM Server instances
+==== Add APM Server or {agent} instances
 
 If the APM Server cannot process data quickly enough,
 you will see request timeouts.
-
 One way to solve this problem is to increase processing power.
-This can be done by either migrating your APM Server to a more powerful machine
+
+**APM Server Binary users**
+
+Increase processing power by either migrating your APM Server to a more powerful machine
 or adding more APM Server instances.
 Having several instances will also increase <<high-availability, availability>>.
 
+**Fleet-managed users**
+
+Increase processing power by either migrating your {agent} to a more powerful machine
+or adding more {agent} instances.
+
```

Comment:
Do we really need two sections here? The action is basically the same. How about something like

> Increase processing power by either migrating to a more powerful machine or adding more APM Server/Elastic Agent instances.